package.json

```diff
@@ -67,7 +67,7 @@
     "karma-babel-preprocessor": "^7.0.0",
     "karma-chrome-launcher": "0.1.4",
     "karma-commonjs": "0.0.8",
-    "karma-coverage": "0.2.6",
+    "karma-coverage": "2.2.0",
```

Comment:
use "karma-coverage": "^2.2.0" So that it picks the highest compatible version.

Reply:
We only want to give support to the exact version as upgrading to higher version may break the support with the node version 10.24.1

Reply:
As I checked, it is safe to use the ^ symbol as it will only upgrade till the minor versions only and after using this I validated in the package-lock.json that we are getting same dependencies with required versions to remove the vulnerabilities.